engine/overworld/movement.asm

```diff
@@ -127,11 +127,7 @@ UpdateNPCSprite:
 	and a
 	jp z, InitializeSpriteStatus
 	call CheckSpriteAvailability
-	; wispnote - Fixes the natural Amazing Man bugs (not the humanly provoked cases).
-	; Also, surprisingly, makes some previously invisble text appear, e.g., the rival's exclamation bubbles during the encounters in Route 22.
-	jp nc, .isAvailableSprite
-	call InitializeSpriteScreenPosition
-	ret; if sprite is invisible, on tile >=$60, in grass or player is currently walking
+	ret c
```

Comment:
needs the comment restored - "if sprite is invisible, on tile >=$60, in grass or player is currently walking"

Reply:
Oops sorry for that. I need to be more careful when I am submiting more than one pull requests.